vg/vgsvg/vgsvg.go

```diff
@@ -461,3 +482,14 @@ func opacityString(clr color.Color) string {
 	_, _, _, a := clr.RGBA()
 	return fmt.Sprintf("%.*g", pr, float64(a)/math.MaxUint16)
 }
+
+// getClassName returns the class name associated to the given style.
+// A different class name is generated for each new style.
+func (c *Canvas) getClassName(style string) string {
```

Comment:
I'd also change the name of this method. Getters in Go are generally spelled without the "get" prefix unless that would make them zero length. I'd call this `className` or preferably `classNameOf`.